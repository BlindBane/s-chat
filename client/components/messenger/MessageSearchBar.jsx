import React from 'react'

const MessageSearchBar = () => {
  return (
    <div>
      <input type='text' placeholder='Seach your messages...' />
    </div>
  )
}

export default MessageSearchBar
