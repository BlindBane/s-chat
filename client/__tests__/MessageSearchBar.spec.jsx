import React from 'react'
import { shallow } from 'enzyme'
import MessageSearchBar from './../components/messenger/MessageSearchBar'

test('MessageSearchBar renders correctly', () => {
  const component = shallow(<MessageSearchBar />)
  expect(component).toMatchSnapshot()
})
