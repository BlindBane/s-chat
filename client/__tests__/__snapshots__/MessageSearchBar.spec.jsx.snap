// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessageSearchBar renders correctly 1`] = `
<div>
  <input
    placeholder="Seach your messages..."
    type="text"
  />
</div>
`;
